src/thunderbots/software/ai/primitive/directwheels_primitive.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include "ai/primitive/primitive.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
+class DirectWheelsPrimitive : public Primitive
+{
+   public:
+    static const std::string PRIMITIVE_NAME;
+    // Power is a fraction of the total power we can apply to the robots,
+    // with +-255 being the max/min, and 0 being no power.
+    /**
+     * Creates a new Move Primitive
+     *
+     * @param robot_id the id of the robot
+     * @param front_left_wheel_power a value between -255 and 255, where positive is
+     * clockwise
+     * @param back_left_wheel_power a value between -255 and 255, where positive is
+     * clockwise
+     * @param front_right_wheel_power a value between -255 and 255, where positive is
+     * clockwise
+     * @param back_right_wheel_power a value between -255 and 255, where positive is
+     * clockwise
```

Comment:
If we want to enforce that the wheel power is between -255/+255, maybe we should use `int16_t` instead? In general it's better if we can enforce things at the type level, rather then at runtime, because it makes it easier for the compiler to catch. In this case it's a bit of a moot point (I don't think using `int16_t` would help the compiler) but at least it's more clear when reading the code what the max/min for `back_left_wheel_power`, `back_right_wheel_power`, etc. are!

Reply:
If we use int16_t, wouldn't we still have to clamp it to the -255/+255 range anyways?

Reply:
That's a good point. It's not going to save us a clamping operation so I don't think it's really necessary.

Reply:
After some discussion on slack, not sure what I was thinking, but `int16_t` definetly doesn't improve this. Looks good as-is!